homeassistant/components/samsungtv/__init__.py

```diff
@@ -252,7 +252,8 @@ async def _async_create_bridge_with_updated_data(
         LOGGER.warning(
             (
                 "Detected model %s for %s. Some televisions from H and J series use "
-                "an encrypted protocol but you are using %s which may not be supported"
+                "an encrypted protocol but you are using %s which may not be supported. "
+                "If your TV is working in the integration, this message can be ignored"
```

Comment:
Logging a warning that can be ignored is not right...
In that case, it ain't a warning; at most informational.

Reply:
> Logging a warning and telling it can be ignored is not right... In that case, it ain't a warning; at most, it is informational in that case.

It was info - but it seems that's not allowed anymore :(

There used to be a lot of queries about these mis-behaving models so the log entry was useful.
I'm worried that switching it back to debug will simply lead to more issues being opened...

Reply:
I think this message comes at the wrong time. It should be a notice in the config flow if this case happens, since that's when the connection method is detected. If the user would be able to do anything about it, we could create a repair issue, but that doesn't seem to be the case.

So I'd remove the log here, and add a step in the config flow just to show this warning if that case happens. Let the user then decide to proceed or abort the flow.